docs/source/docker/advanced/tutorial_al_with_docker.py

```diff
@@ -0,0 +1,144 @@
+"""
+
+.. _docker-active_learning:
+
+Documentation: Use the docker for active learning
+==============================================
+
+For running an active learning step with the Lightly docker, we need to perform
+3 steps:
+
+1. Create an embeddings.csv file with the Lightly docker.
+2. Add your active learning scores as an additonal column.
+3. Use the Lightly docker to perform a sampling on the scores.
+
+
+
+# Step 1:
+Run the docker as usual and as described in the getting started section.
+The only difference is that you set the number of samples to be sampled to 1,
+this prevents sampling.
+E.g. create and run a bash script with the following content:
+
+.. code::
+    INPUT_DIR=/path/to/your/dataset
+    SHARED_DIR=/path/to/shared
+    OUTPUT_DIR=/path/to/output
+
+    TOKEN=YOUR_TOKEN # put your token here
+    N_SAMPLES=1
```

Comment:
If you make this `1.0` instead of `1` we skip sampling as we would select 100% of the dataset